### PR TITLE
Add volcano podgroups permissions

### PR DESCRIPTION
**Pull request by mengfwanaws**
### Description

Spark on Kubernetes with [Volcano](https://volcano.sh/en) as a custom scheduler is supported since Spark v3.3.0 and Volcano v1.5.1.

This PR is to add necessary permissions to support using Volcano as a customized scheduler for EMR Spark on EKS via API [start-job-run](https://docs.aws.amazon.com/cli/latest/reference/emr-containers/start-job-run.html).

The added permissions are similar to that in [spark-operator-rbac.yaml](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/72481c02ebfca1409beba44e07c45a4474626d7d/manifest/spark-operator-install/spark-operator-rbac.yaml#L63-L65). The only difference is that we don't need permissions for Volcano's custom resources `queues` and `queues/status`. This is because we don't need access to the queue, nor shall we manage it. Customers should use the existing `default` queue or create a new queue beforehand if they want to use it.

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory) 
   - We will update https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-cluster-access.html shortly.
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes
- [x] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: pkg/authconfigmap/assets/emr-containers-rbac.yaml [hunk 1]
```diff
@@ -26,7 +26,10 @@ rules:
     verbs: ["get", "list", "watch", "describe", "create", "edit", "delete", "deletecollection", "annotate", "patch", "label"]
   - apiGroups: [""]
     resources: ["persistentvolumeclaims"]
-    verbs: ["create", "list", "delete"]   
+    verbs: ["create", "list", "delete"]
+  - apiGroups: [ "scheduling.volcano.sh" ]
+    resources: [ "podgroups" ]
+    verbs: [ "get", "list", "watch", "create", "delete", "update" ]
```
Comment (cPu1) [inline pkg/authconfigmap/assets/emr-containers-rbac.yaml:32]: Minor nit: the spacing is inconsistent with the other fields.

**Review by cPu1 — APPROVED** [review]

**Comment by mengfwanaws** [comment]
Hi [Himangini](https://github.com/Himangini) and [cPu1](https://github.com/cPu1), may I ask if you want me to fix the nit or you can merge it as it is now? Thanks.

**Comment by Himangini** [comment]
> Hi [Himangini](https://github.com/Himangini) and [cPu1](https://github.com/cPu1), may I ask if you want me to fix the nit or you can merge it as it is now? Thanks.

@mengfwanaws it'll be awesome if you fix the nit for consistency ✨

**Comment by mengfwanaws** [comment]
Hi [Himangini](https://github.com/Himangini) and [cPu1](https://github.com/cPu1), I think I accidentally closed the PR as I squashed my commits. I created a new PR here  -> https://github.com/weaveworks/eksctl/pull/6500

**Pull request closed without merging**